diagnostics/test_canary.md

```diff
@@ -0,0 +1,13 @@
+# Test Feature in Canary
+
+The latest runtime changes are first shipped in the WebView2 runtime in the Edge Canary Browser. If you would like to test the new changes with your application, here's a suggested way to do so:
```

Comment:
nit: "to test the new changes " -> + or help us to verify that your issue is fixed